pkg/apis/kubeone/helpers.go

```diff
@@ -89,6 +91,30 @@ func (crc ContainerRuntimeConfig) String() string {
 	return "unknown"
 }
 
+func (crc *ContainerRuntimeConfig) UnmarshalText(text []byte) error {
```

Comment:
What is the reason for passing the byte slice as a parameter? Can we make it use string instead, especially because the function that uses this helper is using strings?

Reply:
This is conform the https://pkg.go.dev/encoding#TextUnmarshaler interface.